mws/apimws/ipreg.py

```diff
@@ -65,9 +66,17 @@ def delete_cname(hostname):
 
 
 def find_sshfp(hostname):
+    result = []
```

Comment:
Code looks ok but I don't think it provides the same functionality as the code it replaces. The old code checked in the ip_reg database if an sshfp entry existed for the given hostname while the new code checks if the DNS servers have a sshfp entry for the given hostname. The two are different, and before the DNS gets updated (at 53 mins every hour) the former may respond true, and the latter may respond false.

Reply:
For documentation on how the find_sshfp ipreg_api works: https://github.com/uisautomation/mws-misc/blob/f92592afe6ff51423b66119a0cfa0a8c3354e837/dns/API#L142

and the implementation is in

https://github.com/uisautomation/mws-misc/blob/f92592afe6ff51423b66119a0cfa0a8c3354e837/dns/util/perl/ipreg_api.pl#L292